netfields/fields.py

```diff
@@ -56,6 +58,13 @@ def to_python(self, value):
         except ValueError as e:
             raise ValidationError(e)
 
+    def validate(self, value, model_instance):
```

Comment:
no return value?

Reply:
Nope, validate should raise a `ValidationError` in case if an invalid value

Reply:
Cool, Resolved from my side